CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Add an example that specifies the Solidity Coin contract (#576)
  - A work-in-progress example on Solidity's Simple Auction (#573)
 
+### Changed
+
+ - `examples` are grouped by categories (#582)
+
```

Comment:
I don't think we should add changelog entries for doc organization changes. It's not part of the software or api afaiu.